zebra-state/src/service/read/tree.rs

```diff
@@ -50,97 +50,24 @@ where
 
 /// Returns a list of Sapling [`NoteCommitmentSubtree`]s starting at `start_index`.
```

Comment:
There is no `start_index` argument any more.

Please remember to update documentation!

Reply:
Ahh, thank you, fixed in https://github.com/ZcashFoundation/zebra/pull/7739/commits/3416c11264c6c7f4d0cfb3a47429b73f7dde3600.